<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PerfProfiler::Internal::PerfConfigWidget</class>
 <widget class="QWidget" name="PerfProfiler::Internal::PerfConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>500</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1">
    <widget class="QComboBox" name="sampleMode"/>
   </item>
   <item row="3" column="3">
    <widget class="QSpinBox" name="stackSize">
     <property name="maximum">
      <number>65536</number>
     </property>
     <property name="value">
      <number>4096</number>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="stackSizeLabel">
     <property name="text">
      <string>Stack snapshot size (kB):</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="sampleModeLabel">
     <property name="text">
      <string>Sample mode:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="periodLabel">
     <property name="text">
      <string>Sample period:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="callgraphLabel">
     <property name="text">
      <string>Call graph mode:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QLineEdit" name="extraArguments"/>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="callgraphMode">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="extraArgumentsLabel">
     <property name="text">
      <string>Additional arguments:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QSpinBox" name="period">
     <property name="maximum">
      <number>2147483647</number>
     </property>
     <property name="value">
      <number>250</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QTableView" name="eventsView">
     <property name="minimumSize">
      <size>
       <height>300</height>
      </size>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::AllEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="useTracePointsButton">
     <property name="text">
      <string>Use Trace Points</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="addEventButton">
     <property name="text">
      <string>Add Event</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="removeEventButton">
     <property name="text">
      <string>Remove Event</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="resetButton">
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
